dev/reference/error-codes.md

```diff
@@ -18,13 +18,103 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
 | 8003 | `ADMIN CHECK TABLE` 命令在遇到行数据跟索引不一致的时候返回该错误 |
 | 8004 | 单个事务过大，原因及解决方法请参考[这里](/dev/faq/tidb.md#433-transaction-too-large-是什么原因怎么解决) |
 | 8005 | 事务在 TiDB 中遇到了写入冲突，原因及解决方法请参考[这里](/dev/faq/tidb.md#九故障排除) |
+| 8018 | 插件无法重新载入，原因是没有载入过 |
```

Comment:
There are some errors will not report to the users, I didn't list them here. But still, some can't be decided to list here or not. PTAL @bb7133